src/Raven.Server/ServerWide/Commands/Indexes/PutStaggeredIndexCommand.cs

```diff
@@ -0,0 +1,90 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Raven.Client.Documents.Indexes;
+using Raven.Client.ServerWide;
+using Raven.Server.Rachis;
+using Raven.Server.Utils;
+using Sparrow;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Server.ServerWide.Commands.Indexes
+{
+    public class PutStaggeredIndexCommand : UpdateDatabaseCommand
+    {
+        public string IndexName { get; set; }
+
+        public PutStaggeredIndexCommand()
+        {
+            // for deserialization
+        }
+
+        public PutStaggeredIndexCommand(string databaseName, string indexName, string uniqueRequestId) 
+            : base(databaseName, uniqueRequestId)
+        {
+            IndexName = indexName;
+        }
+
+        public override string UpdateDatabaseRecord(DatabaseRecord record, long etag)
+        {
+            if (record.StaggeredIndexes.TryGetValue(IndexName, out var staggeredIndex ) == false)
+            {
+                return null; // was already removed
+            }
+
+            // What happens here though? No memebers.. do we resend the command? How do we make sure this will run again?
```

Comment:
You need to designate someone else, I think.
Alternatively, because we don't modify the database record, the next indexing run will send it again?

Reply:
Yes, the next indexing run will send it again.